mkosi/__init__.py

```diff
@@ -2794,7 +2794,15 @@ def install_debian_or_ubuntu(args: MkosiArgs, root: Path, *, do_run_build_script
             if "VERSION_ID" not in os_release and "BUILD_ID" not in os_release:
                 f.write(f"BUILD_ID=mkosi-{args.release}\n")
 
-    invoke_apt(args, do_run_build_script, root, "install", extra_packages)
+    if args.distribution == Distribution.ubuntu:
+        updates = f"deb http://archive.ubuntu.com/ubuntu {args.release}-updates {' '.join(repos)}\n"
+    else:
+        updates = f"deb http://deb.debian.org/debian {args.release}-updates {' '.join(repos)}\n"
```

Comment:
note that updates != security (although the latter gets synced to the former eventually)

this is the security repo on bullseye (note format changed from this release):

```
deb https://deb.debian.org/debian-security bullseye-security main
```

Reply:
(it's good to enable updates ofc)

Reply:
I figure we can add security as well; is there harm in adding it by default?

Reply:
we should, yes - with two caveats, one the format of the sources line changed in bullseye as mentioned, and two testing/unstable don't have a security repository, so apt update will fail.

So, before bullseye:

`deb http://security.debian.org/debian-security/ buster/updates main`

Bullseye and newer:

`deb https://security.debian.org/debian-security bullseye-security main`

And testing/unstable don't have it (and they also don't have the updates repository)

Reply:
Added security repos to ubuntu/debian as well